blocks/filter/include/gnuradio-4.0/filter/time_domain_filter.hpp

```diff
@@ -36,14 +37,17 @@ H(z) = b[0] + b[1]*z^-1 + b[2]*z^-2 + ... + b[N]*z^-N
     }
 };
 
-enum class IIRForm {
```

Comment:
What's wrong with this enum class definition? I thought this would be a perfectly valid use-case to multiplex on the specific IIR implementation?

Reply:
See the general comment